### PR TITLE
EDUCATOR-5029: Course Staff removing Team submission

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** Allow course staff to remove a team submission from grading 

**What changed?**

- Changed messages to reflect teams / individual
- When cancel_submission is called for a team submission, look up the team submission and cancel the entire team submission.

**Developer Checklist**
- [x] Reviewed the [release process](./release_process.md)
- [x] Translations up to date
- [x] JS minified, SASS compiled
- [x] Test suites passing on Jenkins
- [x] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

JIRA: [EDUCATOR-5029](https://openedx.atlassian.net/browse/EDUCATOR-5029)

FIY: @edx/masters-devs-gta

### CODE REVIEW

File: openassessment/templates/openassessmentblock/staff_area/oa_student_info.html [hunk 1]
```diff
@@ -166,7 +166,11 @@ <h2 class="staff-info__title ui-staff__subtitle">
                 </div>
             {% elif workflow_status == "cancelled" %}
                 <div class="staff-info__final__grade__score">
-                    <p>{% trans "The learner's submission has been removed from peer assessment. The learner receives a grade of zero unless you delete the learner's state for the problem to allow them to resubmit a response." %}</p>
+                    {% if not is_team_assignment %}
+                        <p>{% trans "The learner's submission has been removed from peer assessment. The learner receives a grade of zero unless you delete the learner's state for the problem to allow them to resubmit a response." %}</p>
+                    {% else %}
+                        <p>{% trans "The teams's submission has been removed from peer assessment. The team receives a grade of zero unless you delete the team's state for the problem to allow them to resubmit a response." %}</p>
```
Comment (Andytr1) [inline openassessment/templates/openassessmentblock/staff_area/oa_student_info.html:172]: Team submissions are not part of peer assessments. Perhaps run the wordsmithing by Sapana.

Comment (jansenk) [reply openassessment/templates/openassessmentblock/staff_area/oa_student_info.html:172]: Good catch, copy paste strikes again
